src/static_canvas.class.ts

```diff
@@ -703,8 +703,8 @@ import { pick } from './util/misc/pick';
      * @chainable
      */
     requestRenderAll: function () {
-      if (!this.isRendering) {
-        this.isRendering = fabric.util.requestAnimFrame(this.renderAndResetBound);
+      if (!this.nextRenderHandle && !this.disposed && !this.destroyed) {
```

Comment:
maybe i got confused the first time i read it, this is correct, if i m disposing you can't request more renders.